lib/bundler/source.rb

```diff
@@ -14,6 +14,11 @@ def unmet_deps
       specs.unmet_dependency_names
     end
 
+    def get_source
+      string_repr = to_s
+      string_repr[string_repr.index(/http/)..string_repr.rindex("/")] if string_repr.match(/http/)
+    end
```

Comment:
Can you say more about what this is doing? Is it possible for us to only use `Source#to_s` instead, since we are hashing the URL before adding it to any metrics?

Reply:
In its current state `Source#to_s` will return `#<Bundler::Source::Rubygems:0x47063525993120 rubygems repository https://rubygems.org/ or installed locally`.
I define this methos since I want to hash only `https://rubygems.org/`.

Are you saying I should override `Source#to_s`, or parse the returned object within `metrics.rb` instead of `source.rb`?